Configuration/Geometry/python/dict2023Geometry.py

```diff
@@ -562,7 +562,7 @@
         1 : [
             'Geometry/ForwardCommonData/data/forwardshield/2017/v1/forwardshield.xml',
             'Geometry/ForwardCommonData/data/brmrotations.xml',
-            'Geometry/ForwardCommonData/data/PostLS2/brm.xml',
+            'Geometry/ForwardCommonData/data/brm/2023/v1/brm.xml',

```

Comment:
@bsunanda why did you make this change in F2? This will affect all the Phase2 geometries.